tests/TestCase/Database/Driver/SqlserverTest.php

```diff
@@ -160,6 +158,31 @@ public function testConnectionConfigCustom()
     }
 
     /**
+     * Test if attempting to connect with the driver throws an exception when
+     * using an invalid config setting.
+     *
+     * @expectedException \InvalidArgumentException
+     * @expectedExceptionMessage Sqlserver driver does not allow config setting: persistent
+     * @return void
+     */
+    public function testSettingPersistentException()
+    {
+        $this->skipIf($this->missingExtension, 'pdo_sqlsrv is not installed.');
+        $config = [
+            'persistent' => true,
```

Comment:
Since the 'persistent' option is ignored completely I think this test case would have limited utility.  We don't test that the connection still works with any other unused options (for example option name mispellings).  I can add a test but it feels like overkill.